### DISCUSSION TITLE
not plot my hdd

### DISCUSSION THREAD

**Discussion by 3zar7767** [Chia-Network/chia-blockchain#5594] [Support]
Im try to ploting my external 5tb hdd.
my pc done all step to 100% but after chia wallet not detect my plots and show this massage

![0000](https://user-images.githubusercontent.com/84513951/118986521-4c089180-b934-11eb-85f4-8eb2e056902f.PNG)

**Comment by BrandtH22** [comment]
@3zar7767 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
